api_tutorials/chrom_hmm_cnn.py

```diff
@@ -288,6 +288,8 @@ def bed_file_label(bed_dict, contig, pos):
 
 
 def split_data(datasets, valid_ratio=0.1, test_ratio=0.4):
+	assert(valid_ratio + test_ratio < 1.0)
```

Comment:
while your at it, both should be >0

Reply:
done (I did >=, there could be times when you don't want the 3 way split)...